specs/airr-schema.yaml

```diff
@@ -2259,118 +2253,152 @@ Clone:
+            x-airr:
+                miairr: false
         v_alignment_start:
             type: integer
             description: >
                 Start position in the V segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         v_alignment_end:
             type: integer
             description: >
                 End position in the V segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         d_alignment_start:
             type: integer
             description: >
                 Start position of the D segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         d_alignment_end:
             type: integer
             description: >
                 End position of the D segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         j_alignment_start:
             type: integer
             description: >
                 Start position of the J segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         j_alignment_end:
             type: integer
             description: >
                 End position of the J segment in both the sequence_alignment and germline_alignment fields
                 (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         cdr3_start:
             type: integer
             description: CDR3 start position in the query sequence (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         cdr3_end:
             type: integer
             description: CDR3 end position in the query sequence (1-based closed interval).
+            x-airr:
+                miairr: false
+                required: true
         rearrangement_count:
             type: integer
             description: Number of rearrangements represented by this clone
+            x-airr:
+                miairr: false
         unique_rearrangement_count:
             type: integer
             description: Number of unique rearrangements represented by this clone
+            x-airr:
+                miairr: false
         clone_count:
             type: integer
             description: Number of instances for this clone
+            x-airr:
+                miairr: false
         seed_id:
```

Comment:
How does `clone_count` differ from  `rearrangement_count`?

Reply:
I'm confused about this distinction too. 

I would assume `clone_count` to be at the `Repertoire` level? Ie, the total number of observed clonal families in the Repertoire. That doesn't match the description here though.

Reply:
...and wouldn't belong in the `Clone` object, either

Reply:
I think @schristley added it so maybe could clarify here?

Perhaps it's the number of times this same clone was observed/inferred across repertoires since we talked about wanting to account for the possibility that you might see a clone in different repertoires (hence the emphasis on the discussion of the clone-repertoire relationship)?

Reply:
> How does clone_count differ from rearrangement_count?

I pulled these from immunedb so my understanding might be somewhat limited. `clone_count` is the abundance of that specific clone. It might be as simple as adding up the abundances for all the rearrangements for the clone, or something more complicated. `rearrangement_count` is the number of rearrangement records for that clone, so if `rearrangement_count=7` and you did a query on the rearrangement table for that clone, it should return 7 entries.

Reply:
So it's just the sum of `duplicate_count` for all the rearrangements in the clone? And what about `rearrangement_count` vs `unique_rearrangement_count`?

Given the lack of clarity and the fact that this initial release is supposed to be "minimal," I propose dropping both `unique_rearrangement_count` and `clone_count`, at least for now.

Reply:
@scharch ok with me. Since we allow custom fields, tools like Olmsted (which is where `unique_rearrangement_count` came from I believe) can define these fields for themselves. 

Quantifying the members of a clonal family in different ways seems like it will always application dependent anyway.